source/manual/emergency-publishing.html.md

```diff
@@ -38,8 +38,8 @@ The data for the emergency banner is stored in Redis. Jenkins is used to set the
 1. Go to the Jenkins task:
 
   - [Deploy the emergency banner on Integration](https://deploy.integration.publishing.service.gov.uk/job/deploy-emergency-banner/)
-  - [Deploy the emergency banner on Staging](https://deploy.staging.publishing.service.gov.uk/job/deploy-emergency-banner/)
-  - [⚠️ Deploy the emergency banner on Production ⚠️](https://deploy.publishing.service.gov.uk/job/deploy-emergency-banner/)
+  - [Deploy the emergency banner on Staging](https://deploy.blue.staging.govuk.digital/job/deploy-emergency-banner/)
```

Comment:
Just a minor thing, but I think we're favouring `deploy.staging.govuk.digital` now.

Reply:
I didn't even know that existed! When I try it in my browser I get connection refused, but it could be because I'm on the VPN?

Reply:
As with the `.blue` version, you need to go to `https` directly :-|.

Reply:
Ah yep! I'll update the links.